packages/tests/src/utils/executor.ts

```diff
@@ -49,9 +51,14 @@ export class Executor {
     }
   }
 
-  static login() {
-    const command = `az login --service-principal -u ${process.env.AZURE_CLIENT_ID} -p ${process.env.AZURE_CLIENT_SECRET} -t ${process.env.AZURE_TENANT_ID}`;
-    return this.execute(command, process.cwd());
+  static async login() {
+    const command = `az login -u ${Env["azureAccountName"]} -p '${Env["azureAccountPassword"]}'`;
+    await Executor.execute(command, process.cwd());
```

Comment:
The reason the return value here is false is that no subscription was specified, and I'm not sure if this will have any effect.
![image](https://github.com/OfficeDev/TeamsFx/assets/139844715/c05a903c-5ab3-491a-b4ba-14686281950f)